src/main/java/net/minestom/server/entity/Entity.java

```diff
@@ -133,6 +132,84 @@ public Entity(@NotNull EntityType entityType, @NotNull UUID uuid) {
 
         setAutoViewable(true);
 
+        // Set default gravity values
+        // according to https://minecraft.gamepedia.com/Entity#Motion_of_entities
+        double gravityAcceleration;
```

Comment:
If it depends on the entity type, it might be better to put it in the enum to be easily accessible

Reply:
Okay, moved it in commit 0d2dc80